cli/boilerplate_cli.go

```diff
@@ -113,10 +117,20 @@ func parseOptions(cliContext *cli.Context) (*config.BoilerplateOptions, error) {
 		return nil, err
 	}
 
+	missingKeyAction := config.DEFAULT_MISSING_KEY_ACTION
+	missingKeyActionName := cliContext.String(config.OPT_MISSING_KEY_ACTION)
```

Comment:
Should this be `missingKeyActionValue`? I feel like the name would be `--missing-key-action` or something similar.